### PR TITLE
added gamerherz

### PR DESCRIPTION
**Pull request by Nicouschulas**
## Type of change

- [x ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/gamerherz/background.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background.png:1]: Compress this file.

File: minecraft_servers/gamerherz/background@2x.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress this file.

File: minecraft_servers/gamerherz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "server_name": "gamerherz",
+  "nice_name": "Gamerherz.eu",
+  "direct_ip": "Gamerherz.eu",
+  "server_wildcards": [
+    "%.gamerherz.eu"
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/gamerherz/manifest.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "youtube": "https://www.youtube.com/channel/UCoPAhhSl7Y3sek7qXkQCqWw"
+  },
+  "gamemodes": {
+    "paintball": {
+      "name": "Paintball", 
+      "command": "/quickjoin Paintball-1",
+      "versions": "1.8<*"
+    },
+    "duels 1.8": {
+      "name": "Schach",
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:26]: Key doesn't match name

File: minecraft_servers/gamerherz/manifest.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      "command": "/quickjoin Duels_8-1",
+      "versions": "1.8<*"
+    },
+    "duels 1.16": {
+      "name": "Duels 1.16", 
+      "command": "/quickjoin Duels_16-1",
+      "versions": "1.16<*"
+    },
+    "schach": {
+      "name": "Duels 1.8",
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:36]: Key doesn't match name

File: minecraft_servers/gamerherz/manifest.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "direct_ip": "Gamerherz.eu",
+  "server_wildcards": [
+    "%.gamerherz.eu"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://www.gamerherz.eu/",
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:13]: Had no real content:
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/manifest.json [hunk 5]
```diff
@@ -2,15 +2,13 @@
...
-  ],
+
   "supported_languages": [
-     "de",
-     "en"
+     "de"
   ],
   "social": {
-    "web": "https://www.gamerherz.eu/",
+    "web": "https://gamerherz.eu",
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:10]: No valid content

File: minecraft_servers/gamerherz/manifest.json [hunk 6]
```diff
@@ -2,15 +2,13 @@
...
+    "web_support": "https://gamerherz.eu/contact.html
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:11]:
```suggestion
    "web_support": "https://gamerherz.eu/contact.html",
```

File: minecraft_servers/gamerherz/manifest.json [hunk 7]
```diff
@@ -2,15 +2,13 @@
   "server_name": "gamerherz",
   "nice_name": "Gamerherz.eu",
   "direct_ip": "Gamerherz.eu",
-  "server_wildcards": [
-    "%.gamerherz.eu"
-  ],
+
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:5]:
```suggestion
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/background.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background.png:1]: Compress this file

File: minecraft_servers/gamerherz/background@2x.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress this file

File: minecraft_servers/gamerherz/manifest.json [hunk 8]
```diff
@@ -0,0 +1,39 @@
+{
+  "server_name": "gamerherz",
+  "nice_name": "Gamerherz.eu",
+  "direct_ip": "Gamerherz.eu"
+
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:5]:
```suggestion
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/manifest.json [hunk 9]
```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "gamerherz",
+  "nice_name": "Gamerherz.eu",
+  "direct_ip": "Gamerherz.eu"
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:4]:
```suggestion
  "direct_ip": "gamerherz.eu",
```

File: minecraft_servers/gamerherz/background.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background.png:1]: Compress this file.

File: minecraft_servers/gamerherz/background@2x.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress this file.

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/background.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background.png:1]: Compress

File: minecraft_servers/gamerherz/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**